VAMobile/src/screens/HealthScreen/SecureMessaging/StartNewMessage/StartNewMessage.test.tsx

```diff
@@ -50,6 +50,12 @@ jest.mock('store/slices', () => {
         payload: '',
       }
     }),
+    resetSaveDraftComplete: jest.fn(() => {
+      return {
```

Comment:
Had to update this so the unit tests worked once I deleted the claims and appeals slice. no idea why it was working before deletion, but whatever lol